src/UXCapture.js

```diff
@@ -47,8 +80,22 @@ const UXCapture = {
 		_view.update(zoneConfigs);
 	},
 
-	// TODO: SPA support in subsequent ticket
-	startTransition: () => {},
+	startTransition: () => {
+		UXCapture._clearMarksAndMeasures();
+
+		if (
+			typeof window.performance !== 'undefined' &&
+			typeof window.performance.mark !== 'undefined'
+		) {
+			window.performance.mark(INTERACTIVE_TRANSITION_START_MARK_NAME);
+		}
+		_startMarkName = INTERACTIVE_TRANSITION_START_MARK_NAME;
+
+		// reset the view until it's defined again using startView();
+		_view = undefined;
+
+		ExpectedMark.clearExpectedMarks();
```

Comment:
should this be handled in `UXCapture._clearMarksAndMeasures();`? It's a little confusing to have the two separate 'clearing' calls

Reply:
Good catch, it looks like one can call another. I'll probably also rename `ExpectedMark.clearExpectedMarks();` to `ExpectedMark.clearExpectedMarksMap();` to make it clear that it clears internal structure rather than performance timeline.